# request a blob (left, right, middle, largest)

string type
---
cmvision/Blob blob
